src/style/reset.css

```diff
@@ -123,3 +123,7 @@ table {
   border-collapse: collapse;
   border-spacing: 0;
 }
+p:empty,
```

Comment:
I need to steal this for all my projects. Makes so much sense.

Reply:
it's great right???